phpBB/docs/events.md

```diff
@@ -1375,6 +1375,20 @@ overall_header_head_append
 * Since: 3.1.0-a1
 * Purpose: Add asset calls directly before the `</head>` tag
 
+overall_header_headerbar_before
+===
+* Locations:
+    + styles/prosilver/template/overall_header.html
+* Since: 3.1.10-RC1
+* Purpose: Add content at the beginning of the headerbar
+* 
+overall_header_headerbar_after
```

Comment:
Events are sorted alphabetically which means the _after event needs to be before the _before event in this document.